tutorials/Homology-Modelling/homology-modelling-tutorial.ipynb

```diff
@@ -237,7 +251,7 @@
    "source": [
     "- Once you are satisfied with your alignment you can focus on generating a 3D model.  Go to [rcsb](http://www.rcsb.org) and search for 6pv7.\n",
     "    \n",
-    "- Download a .pdb file of 6pv7 to your current working directory, then open 6pv7.pdb in pymol.\n",
+    "- Download a .pdb file of 6pv7 to your current working directory, then open 6pv7.pdb in pymol.  This is the PDB of the alpha3beta4 nicotinic receptor (see Gahrpure et al 2019 - https://doi.org/10.1016/j.neuron.2019.07.030), which we will use as a template (other structures could be used - indeed, if there are potentially different templates, you should consider which one might be best or even using more than one at the same time - for now lets just assume that this our template of choice).\n",
```

Comment:
This quite a lot to read in a single bullet point. Maybe worth breaking into a sub-bullet point?